docs/tutorials/hobby_calibration.rst

```diff
@@ -27,32 +29,32 @@ Softare and Firmware Setup
 To follow this tutorial, ensure that your **IQ Control Center is on version 1.2.6 or later**. Earlier versions lacked some of the calibration features that will be discussed. If you
 are unfamiliar with the Control Center, refer to :ref:`control_center_tutorial` for basic usage information. 
 
-If using a Vertiq 2306, it should be on **firmware version 26 or later**. If using a Vertiq 8108, it should be on **firmware version 5 or later**. Earlier versions of the firmware for these
+If using a Vertiq 2306, it should be on **firmware version v0.0.26 or later**. If using a Vertiq 8108, it should be on **firmware version 0.0.5 or later**. Earlier versions of the firmware for these
```

Comment:
since we've released the 40, should we mention it here? We'll also need to make sure this stays updated with new releases that support the hobby stuff